src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,18 @@
+$main-color: rgb(182, 33, 33);
+$add-color: rgb(245, 84, 84);
+$bacground-color: rgb(248, 244, 226);
+
+$container-size: 300px;
+$point-size: 20px;
+
+$point-margin: ($container-size - $point-size) / 2;
+
+$minutes-width: 16px;
+$minutes-height: 100px;
+$minutes-margin-top: $container-size/ 2 - $minutes-height;
+$minutes-margin-left: ($container-size - $minutes-width) / 2;
```

Comment:
I would recommend using a function to calculate these variables

Reply:
These are two different formulas. I think it is not necessary to create complicated forms.